app/Http/Controllers/Api/PostsController.php

```diff
@@ -121,6 +116,6 @@ public function reviews(Request $request)
         $meta = [];
 
         // @TODO: we'll need to change the transformer here depending on what type of post.
-        return $this->collection($reviewedPosts, $code, $meta, $this->photoTransformer);
+        return $this->item($reviewedPost, $code, $meta, $this->photoTransformer);
```

Comment:
wait sorry - I am totally confusing myself. It should go through to the Post Controller but it looks like it skips and goes right to the Post Service. I will fix this!